sdk/vaa/structs.go

```diff
@@ -427,8 +422,6 @@ const (
 	ChainIDNear ChainID = 15
 	// ChainIDMoonbeam is the ChainID of Moonbeam
 	ChainIDMoonbeam ChainID = 16
-	// ChainIDNeon is the ChainID of Neon
-	ChainIDNeon ChainID = 17
```

Comment:
You might add a comment here as well for the obsolete chain id 17.

Reply:
Added a comment saying it's obsolete.